Makefile

```diff
@@ -185,6 +185,11 @@ destroy-full-local-setup: ## Destroy full local multicluster setup
 	k3d cluster delete $(CLUSTER_GSLB2)
 	docker network rm $(CLUSTER_GSLB_NETWORK)
 
+.PHONY: deploy-prometheus
+deploy-prometheus:
```

Comment:
@kuritka do you plan to add the possibility to uninstall Prometheus from the local clusters (`make uninstall-promethus` target) as well? If we install something, there should be a similar way to uninstall it.

Reply:
Makes sense to update contributor's documentation with that target

Reply:
@somaritane, all your comments implemented.